docs/atari/flag_capture.md

```diff
@@ -47,6 +47,23 @@ narrow down the location of the flag quickly and effectively.
 
 Environment parameters are common to all Atari environments and are described in the [base Atari documentation](../atari) .
 
+### Action Space (Minimal)
+
+In any given turn, an agent can choose from one of 10 actions.
+
+| Action    | Behavior  |
+|:---------:|-----------|
```

Comment:
I think this one looks good https://github.com/Farama-Foundation/Multi-Agent-ALE/blob/master/src/games/supported/FlagCapture.cpp#L65